packages/modules/manage-versions/src/components/CreateUpdateVersion/CreateVersionModal/CreateVersionModal.scss

```
@@ -2,21 +2,18 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-@import "~@itwin/itwinui-css/scss/variables";
 
 .iac-additional-info {
   display: flex;
   justify-content: space-between;
-  padding: $iui-baseline $iui-sm;
-  @include themed {
-    background-color: t(iui-color-background-2);
-  }
+  padding: var(--iui-size-s);
+  background-color: var(--iui-color-background-2);
```

`--iui-color-background-2` no longer exists, I'm not sure what would be the best replacement, probably simply go with `--iui-color-background-zebra`
```suggestion
  background-color: var(--iui-color-background-zebra);
```

---

Done